<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Weld</class>
 <widget class="QWidget" name="Weld">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>194</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>520</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>焊接参数设置</string>
  </property>
  <widget class="QLabel" name="weldingVoltage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>焊接电压：</string>
   </property>
  </widget>
  <widget class="QLabel" name="weldingCurrent">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>焊接电流：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="weldVoltageLineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>25.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="weldCurrentLineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>100.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="unit5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>伏（V)</string>
   </property>
  </widget>
  <widget class="QLabel" name="unit6">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>安（A)</string>
   </property>
  </widget>
  <widget class="QLabel" name="wireFeedRateTextLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>送丝速度：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="wireFeedRateLineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="unit4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>米/分（m/min)</string>
   </property>
  </widget>
  <widget class="QLabel" name="gasFlowTextLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>保护气流量：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sheildingGasFlowLineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="unit3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>升/分（L/min)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmPushButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPushButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="weldingRateTextLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>25</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>焊接速度：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="weldSpeedLineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="unit1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>毫米（mm/s)</string>
   </property>
  </widget>
  <widget class="QLabel" name="trackingDistanceLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>焊接长度：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="trackingDistanceLineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>100.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="unit2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>毫米（mm)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
